build.gradle.kts

```diff
@@ -62,7 +63,7 @@ kover.xmlReport {
 
 tasks {
     wrapper {
-        gradleVersion = properties("gradleVersion")
+        gradleVersion = properties("gradleVersion").get()
```

Comment:
is this necessary? wouldn't it cause this value to be eagerly retrieved?

Reply:
It would but the `Wrapper` task doesn't support lazy configuration yet.
By evaluating this eagerly this makes the `gradleVersion` Gradle Property an input to the build configuration. Which sounds fine.

Reply:
Unfortunately, the `Wrapper` task is not yet migrated to the provider api.